### PR TITLE
more tests

### PR DESCRIPTION
**Pull request by sdelamo**

### CODE REVIEW

File: src/main/resources/application-gcp.properties
Comment (timyates) [inline src/main/resources/application-gcp.properties:1]: Where's this from?

**Review by timyates — APPROVED** [review]

File: src/main/resources/application-gcp.properties
Comment (sdelamo) [reply src/main/resources/application-gcp.properties:1]: what do you mean?

File: src/main/java/io/micronaut/starter/analytics/postgres/gcp/IP_TYPE.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.starter.analytics.postgres.gcp;
+
+/**
+ * IP type for connecting to a Cloud SQL instance.
+ */
+public enum IP_TYPE {
```
Comment (timyates) [inline src/main/java/io/micronaut/starter/analytics/postgres/gcp/IP_TYPE.java:21]: Why is this named weirdly?

File: src/main/resources/application-gcp.properties
Comment (timyates) [reply src/main/resources/application-gcp.properties:1]: I can't find this in starter (where this is from)?

**Pull request merged**